po-defs/Makefile.am

```diff
@@ -2,9 +2,13 @@
 # po-defs/Makefile.am for qalculate
 #
 
-noinst_PROGRAMS = fixpo
+LINK_FOR_BUILD.cpp = $(CXX_FOR_BUILD) $(CXXFLAGS_FOR_BUILD) $(CPPFLAGS_FOR_BUILD) $(TARGET_ARCH_FOR_BUILD) $(LDFLAGS_FOR_BUILD)
 
-fixpo_SOURCES = remove-untranslated.cc
+all: fixpo$(BUILD_EXEEXT)
+install: fixpo$(BUILD_EXEEXT)
```

Comment:
Had this be a prereq of `install` since otherwise the `install` path wouldn't build it, but it would still try to run `fixpo` on the po files (and I think it did that before when it was `noinst_PROGRAMS` too).